--- src/export/generatePptx.ts
+++ src/export/generatePptx.ts
@@ -367,3 +367,3 @@
           
-          slide.addShape((pres.ShapeType && pres.ShapeType.customGeometry) || 'customGeometry', pathOpts)
+          slide.addShape(((pres.ShapeType as any).customGeometry || 'customGeometry'), pathOpts)
           continue
--- src/export/generateSvg.ts
+++ src/export/generateSvg.ts
@@ -131,3 +131,3 @@
   for (const m of match) {
-    fonts.add(m[1].split(',')[0].trim().replace(/['"]/g, ''))
+    fonts.add(m[1]!.split(',')[0]!.trim().replace(/['"]/g, ''))
   }
